src/components/Autocomplete/components/ComboBox/ComboBox.tsx

```diff
@@ -470,19 +472,22 @@ export default class ComboBox extends React.PureComponent<Props, State> {
 
   @autobind
   private selectOptionAtIndex(newOptionIndex: number) {
-    const {navigableOptions} = this.state;
+    const {navigableOptions, selectedOption} = this.state;
```

Comment:
```suggestion
    const {navigableOptions, selectedOption: oldSelectedOption} = this.state;
```

Reply:
Sorry missed this comment. On 475 your deconstructing selectedOptions, and 480 your just renaming it. I suggested a named deconstruction so we can get rid of line 480 😄